auth-api/src/auth_api/services/keycloak.py

```diff
@@ -222,6 +222,9 @@ def remove_from_account_holders_group(keycloak_guid: str = None, **kwargs):
         """Remove user from the group."""
         if not keycloak_guid:
             user_from_context: UserContext = kwargs['user_context']
+            # check if the role 'account_holder' is present.
+            if Role.ACCOUNT_HOLDER.value not in user_from_context.roles:
+                raise BusinessException(Error.MEMBERSHIP_NOT_FOUND, None)
```

Comment:
This is still going to cause an error when removing staff. They don't have `Role.ACCOUNT_HOLDER`

Reply:
Remove, see below